src/layouts/hass-subpage.ts

```diff
@@ -53,9 +53,9 @@ class HassSubpage extends LitElement {
         height: 64px;
         padding: 0 16px;
         pointer-events: none;
-        background-color: var(--primary-color);
+        background-color: var(--subpage-toolbar-background-color);
```

Comment:
Just use the `app-header-` variables instead of creating new ones.

Reply:
Done!